app/helpers/data_getter.py

```diff
@@ -98,8 +98,8 @@ def get_invite_by_user_id(user_id):
     @staticmethod
     def get_all_events():
         """Method return all events"""
-        return Event.query.order_by(desc(Event.id)).filter_by(in_trash=False).all()
-
+        return Event.query.order_by(desc(Event.created_at)) .filter_by(in_trash=False).all()
```

Comment:
Please remove the spaces. I doubt line 101 even works. Please check code before pushing.